README.md

```diff
@@ -192,22 +192,112 @@ You can use the endpoints to make API requests that read and write data from
 the `ECommerce` database.
 
 For example, with the local server running in a separate terminal tab, run the
-following curl request to the `POST /customers` endpoint. The request creates a
-`Customer` collection document in the `ECommerce` database.
+following curl request to the `POST /products` endpoint. The request creates a
+`Product` collection document in the `ECommerce` database.
 
-```
+```sh
 curl -v \
-  http://localhost:8000/customers \
+  http://localhost:8000/products \
   -H "Content-Type: application/json" \
   -d '{
-    "name": "John Doe",
-    "email": "john.doe@example.com",
-    "address": {
-      "street": "123 Main St",
-      "city": "San Francisco",
-      "state": "CA",
-      "postalCode": "12345",
-      "country": "USA"
-    }
+    "name": "The Old Man and the Sea",
+    "price": 899,
+    "description": "A book by Ernest Hemingway",
+    "stock": 10,
+    "category": "books"
   }'
 ```
+
+
+## Expand the app
+
+You can further expand the app by adding fields and endpoints.
+
+As an example, the following steps adds a computed `totalPurchaseAmt` field to
+Customer documents and related API responses:
+
+1. If you haven't already, add the sample data:
+
+    ```sh
+    npm install && npm run test
+    ```
+
+1. In `schema/collections.fsl`, add the following `totalPurchaseAmt` computed
```

Comment:
Should this be `2. In ...`? Same for the list entries below. Otherwise looks good to me!

Reply:
GitHub handles the counting for me! You can check out the preview here: https://github.com/fauna/js-sample-app/blob/expand-app/README.md

I tend to just use `1.` so I can move items around without changing the nums. I can change this if it's a big deal tho.

Reply:
Just pushed [`f6e3311` (#45)](https://github.com/fauna/js-sample-app/pull/45/commits/f6e3311f0cf3d084969e071339de41760d1ae027) to use regular numbering. That'll be easier if someone is looking at this locally.